### PR TITLE
updated text-recognization file 

### PR DESCRIPTION
**Pull request by vivektangudu2024**
used isbn number of book to print Author name and publisher 
from the text recognized by natik

### CODE REVIEW

**Comment by GrayFlash** [comment]
Hey nice!! Btw move your file to a directory named `PyScripts`, and will be great if you could add a demo on the data that can be obtained.

**Comment by vivektangudu2024** [comment]
i tested on this

**Comment by vivektangudu2024** [comment]
test

**Comment by vivektangudu2024** [comment]
py

**Comment by vivektangudu2024** [comment]
this is the output

File: Pyscripts/Text_recognization.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#pip install easyocr
+
+#pip install isbntools
```
Comment (GrayFlash) [inline Pyscripts/Text_recognization.py]: can you update the requirements.txt file with these pip packages?? Also remove these from the final file.

File: Pyscripts/Text_recognization.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#pip install easyocr
+
+#pip install isbntools
+
+import easyocr
+
```
Comment (GrayFlash) [inline Pyscripts/Text_recognization.py]: no need of this extra line between two imports

File: Pyscripts/Text_recognization.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#pip install easyocr
+
+#pip install isbntools
+
+import easyocr
+
+from isbntools.app import *
+
+reader = easyocr.Reader(['en'])
```
Comment (GrayFlash) [inline Pyscripts/Text_recognization.py]: we can create a function here?? it should accept `imagePath` here, and then we can perform the ISBN reading from image??

File: Pyscripts/Text_recognization.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+from isbntools.app import *
+
+reader = easyocr.Reader(['en'])
+output = reader.readtext('test.jpg')
+final_text=""
+for i in range(0,len(output)):
+    text=output[i][1]
+    final_text=final_text +" "+ text
+    #print(text)
```
Comment (GrayFlash) [inline Pyscripts/Text_recognization.py]: remove commented logging statements

File: Pyscripts/Text_recognization.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+#print(final_text)
+get_isbn = isbn_from_words(final_text)
+
+
+
+print(registry.bibformatters['labels'](meta(get_isbn)))
```
Comment (GrayFlash) [inline Pyscripts/Text_recognization.py]: remove multiple blank lines and use a return statement here, possibly return a dictionary.

Comment (GrayFlash) [reply Pyscripts/Text_recognization.py]: also leave a blank line at the end of files.

**Review by GrayFlash — CHANGES_REQUESTED** [review]

**Comment by vivektangudu2024** [comment]
py2

**Comment by vivektangudu2024** [comment]
these are the keys respectively

**Review by GrayFlash — APPROVED** [review]
LGTM!!

**Pull request merged**